content/en/blog/_posts/2024-04-05-diy-create-your-own-cloud-with-kubernetes-part-1/index.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: blog
+title: "DIY: Create Your Own Cloud with Kubernetes (Part 1)"
+slug: diy-create-your-own-cloud-with-kubernetes-part-1
+date: 2024-04-05
```

Comment:
Let's pick a particular time during this day for the article to publish. They ought to be at least 5 minutes apart.

Reply:
Sure, I'd like to publish this already soon. Is there anything depends on me?

What should I do?

Reply:
How about `2024-04-05T07:30:00+00:00` for this article, and the follow ups 5 minutes apart?